packages/amplify-graphql-schema-generator/src/schema-generator/schema-overrides.ts

```diff
@@ -0,0 +1,67 @@
+import { DocumentNode, FieldDefinitionNode, ObjectTypeDefinitionNode, visit } from 'graphql';
+import { isArrayOrObject, findMatchingField, getNonModelTypes, isOfType, isNonNullType } from 'graphql-transformer-common';
+import { printer } from '@aws-amplify/amplify-prompts';
+
+export const applySchemaOverrides = (document: DocumentNode, existingDocument: DocumentNode): DocumentNode => {
```

Comment:
I see that we are importing the schema then applying the overrides using existing schema. This may rearrange the types in the schema document and also customer would loose their comments. Instead can we mutate the existing schema?

Not a strong requirement but it would be a good DX.

Reply:
Metadata like comments are not available in the GraphQL AST. [Refer](https://github.com/graphql/graphql-js/issues/2241). The ordering of types is dictated by `print` which sorts the types in ascending order to be consistent.